package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "@reactioncommerce/api-plugin-accounts": "^1.3.0",
```

Comment:
@loan-laux should this be updated to https://github.com/reactioncommerce/api-plugin-accounts/releases/tag/v1.4.0 now that the plugin has been published?

Reply:
Yeah we need v1.4.0. I'll update the PR.

Reply:
Oh wait I can't because it's not in our Github org. Duh!